src/server/services/toeristische-verhuur.ts

```diff
@@ -23,10 +39,86 @@ export interface ToeristischeVerhuurRegistratie {
   street: string | null;
 }
 
+export interface TransformedVakantieverhuur extends Vakantieverhuur {
```

Comment:
Een gedachte experiment. Hoe zouden we dit anders kunnen benoemen? Deze logica dicteert:

```
interface TransformedTransformedVakantieverhuur extends TransformedVakantieverhuur { ... }
```

In z'n  benaming niet echt schaalbaar.

Reply:
Misschien: `import { VakantieVerhuur as VakantieVerhuurVergunning } from ....` ?